### ISSUE TITLE
x/website: golang.org/doc/articles/wiki is broken

### ISSUE THREAD

**Issue by kortschak** [golang/go#44314]
The https://golang.org/doc/articles/wiki/ page linked from https://golang.org/doc/#articles is broken presumably due to the content being moved out to x/website in 0cb3415154ff354b42db1d65073e9be71abcc970.

![Screenshot from 2021-02-17 10-41-44](https://user-images.githubusercontent.com/275221/108137896-c5fa6600-710c-11eb-9a42-a4bd47aeb183.png)

**Comment by ianlancetaylor** [comment]
CC @rsc @dmitshur

**Comment by dmitshur** [comment]
The problem is because the wiki directory in x/website contains a file named "go.mod":

https://github.com/golang/website/blob/master/_content/doc/articles/wiki/go.mod

Despite being inside a _content directory that the go tool largely ignores due to the "_" prefix, it isn't ignored for the purpose of carving out a nested module. The content of that directory is not included in [`website.Content`](https://github.com/golang/website/blob/07a4af9ca989df24a90f3c875e02cd1031920e50/content.go#L16), because `embed.FS` doesn't reach across module boundaries.

**Comment by dmitshur** [comment]
I'm not sure what's the best thing to do here. If we continue to use `//go:embed`, we can't place any files named "go.mod" in _content. Maybe we shouldn't do that regardless.

I've drafted [CL 292829](https://golang.org/cl/292829) with a temporary workaround of reading files from disk (as was done before [CL 291690](https://golang.org/cl/291690). We can consider using that approach to fix the immediate problem on golang.org and buy more time to think about a long term fix here. It's not quite finished, I just sent it to share the idea since I saw Russ is also looking into this at the same time.
